src/SDKs/Network/Network.Tests/Network.Tests.csproj

```diff
@@ -39,5 +39,6 @@
   <ItemGroup>
     <Folder Include="SessionRecords\Network.Tests.Tests.AvailableProvidersListTests\" />
     <Folder Include="SessionRecords\Network.Tests.Tests.AzureReachabilityReportTests\" />
+    <Folder Include="SessionRecords\Network.Tests.Tests.ConnectionMonitorTests\" />
```

Comment:
remove lines 39-43
you do not need this explicitly, line 24-26 includes all directories

Reply:
Done